Core/Src/canardRxQueue.c

```diff
@@ -0,0 +1,47 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <canard.h>
+#include "canardRxQueue.h"
+
+struct CanardRxQueueItem* rxQueueHead = NULL;
+struct CanardRxQueueItem* rxQueueTail = NULL;
+
+void enqueueRxFrame(CanardCANFrame* rx_frame) {
```

Comment:
Currently, this enqueue function returns nothing. It would be so much better if this function could return some sort of error code. For instance, 
` enum rxQueueErrorCode{
      SUCCESS,
      EMPTYITEM,
      OVERFLOW
}`